src/ToggleGroup/ToggleGroup.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
A disabled prop is show in storybook, but it doesn't do anything.

Reply:
There was an unneeded prop in the StyleProps of ToggleGroup. The controls is driven automatically from the code so it was pulling through here